### PR TITLE
geonkick: 2.3.8 -> 2.4.0

### PR DESCRIPTION
**Pull request by magnetophon**
###### Things done

- [x] Tested using sandboxing ([nix.useSandbox](https://nixos.org/nixos/manual/options.html#opt-nix.useSandbox) on NixOS, or option `sandbox` in [`nix.conf`](https://nixos.org/nix/manual/#sec-conf-file) on non-NixOS linux)
- Built on platform(s)
   - [x] NixOS
   - [ ] macOS
   - [ ] other Linux distributions
- [ ] Tested via one or more NixOS test(s) if existing and applicable for the change (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
- [ ] Tested compilation of all pkgs that depend on this change using `nix-shell -p nixpkgs-review --run "nixpkgs-review wip"`
- [x] Tested execution of all binary files (usually in `./result/bin/`)
- [ ] Determined the impact on package closure size (by running `nix path-info -S` before and after)
- [ ] Ensured that relevant documentation is up to date
- [x] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/.github/CONTRIBUTING.md).

### CODE REVIEW

**Comment by SuperSandro2000** [comment]
Can you use ``with stdenv.lib;`` in your meta sections?

**Comment by magnetophon** [comment]
@SuperSandro2000 Done.

File: pkgs/applications/audio/geonkick/default.nix [hunk 1]
```diff
@@ -1,28 +1,30 @@
...
   };
 
   nativeBuildInputs = [ cmake pkg-config ];
 
   buildInputs = [ redkite libsndfile rapidjson libjack2 lv2 libX11 cairo ];
 
   # https://github.com/iurie-sw/geonkick/issues/120
-  cmakeFlags = [ "-DGKICK_REDKITE_SDK_PATH=${redkite}" "-DCMAKE_INSTALL_LIBDIR=lib" ];
+  cmakeFlags =
+    [ "-DGKICK_REDKITE_SDK_PATH=${redkite}" "-DCMAKE_INSTALL_LIBDIR=lib" ];
```
Comment (SuperSandro2000) [inline pkgs/applications/audio/geonkick/default.nix:21]:
```suggestion
  cmakeFlags = [
    "-DGKICK_REDKITE_SDK_PATH=${redkite}"
    "-DCMAKE_INSTALL_LIBDIR=lib"
  ];
```

Comment (magnetophon) [reply pkgs/applications/audio/geonkick/default.nix]: Done.

**Review by SuperSandro2000 — APPROVED** [review]
LGTM

**Pull request merged**